assets/css/main.scss

```diff
@@ -1,6 +1,6 @@
 html {
   font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;
-  font-size: 18px;
+  font-size: 18px !important;
```

Comment:
Vuetify has decided to include this style internally... this means there is likely a configuration option for it that we need to find.